acceptance/tests/ingress-gateway/ingress_gateway_test.go

```diff
@@ -18,24 +18,17 @@ const StaticClientName = "static-client"
 // Test that ingress gateways work in a default installation and a secure installation.
 func TestIngressGateway(t *testing.T) {
 	cases := []struct {
-		secure      bool
-		autoEncrypt bool
+		secure bool
 	}{
 		{
 			false,
-			false,
-		},
-		{
-			true,
-			false,
 		},
 		{
 			true,
-			true,
 		},
```

Comment:
IDC either way, but in your other PR, I remember you changed the struct values to be named (e.g. `{ secure: true }`).

Reply:
that was the other file. but i can make it in this file as well.